spec/support/activity_helpers.rb

```diff
@@ -0,0 +1,18 @@
+module ActivityHelpers
+  def page_displays_an_activity(activity_presenter:)
```

Comment:
We've got a mixture of helpers and shared examples now, do you think this is OK? Or that we should go with one pattern or the other? I'm easy either way.

Reply:
I think helpers are good we need to reuse the way we are asserting across different (but similar) contexts. I think shared examples are useful when you can defer the responsibility for the whole setup, execution and assertions.

In the past I've tried to send variables into shared_examples for the setup and exectute steps to make the examples reusable but never ended up feeling happy that they've made things better.

For example in this use of the helper the way the user arrives on that page changes based on the context setup by the feature https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/95bcd0fd1b794adb8111382ea70c2bc1470c84f0/spec/features/staff/users_can_view_programme_level_activity_spec.rb#L21 I'd like to use shared examples if we can apply them in a case like this?

Reply:
Yes that makes sense to me!